cmd/gpud/login/command.go

```diff
@@ -192,6 +192,12 @@ func Command(cliContext *cli.Context) error {
 	}
 	log.Logger.Debugw("successfully got fifo file")
 
+	log.Logger.Debugw("recording control plane login success")
+	if err := pkgmetadata.SetMetadata(rootCtx, dbRW, pkgmetadata.MetadataKeyControlPlaneLoginSuccess, fmt.Sprintf("%d", time.Now().Unix())); err != nil {
```

Comment:
when gpud logout, do we remove these information?

Reply:
We don't. We don't really support logout via cli for now.

Reply:
Yes, we can purge all these metadata, when we support logout